apps/gauzy/src/app/pages/dashboard/team/all-team/all-team.component.html

```
@@ -92,44 +92,31 @@ <h6 class="title">{{ 'TIMESHEET.WORKED_FOR_DAY' | translate }}</h6>
-								('DASHBOARD_PAGE.CHARTS.NOT_WORKING' | translate) +
-								' ' +
-								team?.statistics?.countNotWorking
-							"
-              ></nb-badge>
+              <span class="legend-label">{{ 'DASHBOARD_PAGE.CHARTS.NOT_WORKING' | translate }}:</span>
+              <nb-badge status="danger" [text]="'' + team?.statistics?.countNotWorking"></nb-badge>
             </div>
           </div>
           <div>
-            <button (click)="onSelectTeam(team)" nbButton outline size="small" status="primary">
+            <button (click)="onSelectTeam(team)" nbButton ghost size="small" status="primary">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)